os/fs/procfs/fs_procfsproc.c

```diff
@@ -256,6 +266,12 @@ static const struct proc_node_s g_groupfd = {
 	"group/fd", "fd", (uint8_t)PROC_GROUP_FD, DTYPE_FILE	/* Group file descriptors */
 };
 
+#if defined(CONFIG_PM) && defined(CONFIG_PM_METRICS)
+static const struct proc_node_s g_pminfo = {
+	"pminfo", "pminfo", (uint8_t)PROC_PMINFO, DTYPE_FILE	/* Group file descriptors */
```

Comment:
Please change the description. This is not for group file descriptors.

Reply:
modified.